modules/administration-guide/pages/removing-user-data.adoc

```diff
@@ -9,141 +9,45 @@
 
 == Removing user data according to GDPR
 
-The General Data Protection Regulation (link:https://en.wikipedia.org/wiki/General_Data_Protection_Regulation[GDPR]) law enforces the right for individuals to have personal data erased.
 
-The following procedure describes how to remove the {prod-short} user’s data.
-
-[NOTE]
-====
-The following commands use the default {platforms-namespace}, `{prod-namespace}`, as a user's example for the `-n` option.
-====
+You can remove the {prod-short} user’s data using the {prod-short} API.
+Following this procedure makes the service compliant to EU General Data Protection Regulation (link:https://en.wikipedia.org/wiki/General_Data_Protection_Regulation[GDPR]) that enforces the right for individuals to have personal data erased.
```

Comment:
This procedure makes the service compliant with EU General Data Protection Regulation (link:https://en.wikipedia.org/wiki/General_Data_Protection_Regulation[GDPR]) which enforces the right of individuals to have personal data erased.